convert/convert.go

```diff
@@ -195,3 +210,29 @@ func removeServiceName(service *file.FService) *file.FService {
 	serviceCopy.ID = kong.String(utils.UUID())
 	return serviceCopy
 }
+
+func convertDistributedToKong3(
```

Comment:
nit for consistency with existing code:

```suggestion
func convertDistributedToKong3x(
```